java2typescript-jackson/src/test/resources/java2typescript/jackson/module/WriterPreferencesTest.enumToStringLiteralType.d.ts

```diff
@@ -3,5 +3,11 @@ export type Enum =
     | "VAL2"
     | "name";
 
+export interface TestClassHasMapWithEnumKey {
+    enumKeyMap: { [key in Enum ]: string;};
```

Comment:
nice, i thought i tried adding `?`, but it first seemed to me that it didn't work - probably i added it before `]` not after